Lib/unittest/test/testmock/testmagicmethods.py

```diff
@@ -286,6 +287,8 @@ def test_magicmock_defaults(self):
         self.assertEqual(math.trunc(mock), mock.__trunc__())
         self.assertEqual(math.floor(mock), mock.__floor__())
         self.assertEqual(math.ceil(mock), mock.__ceil__())
+        self.assertTrue(asyncio.iscoroutinefunction(mock.__aexit__))
+        self.assertTrue(asyncio.iscoroutinefunction(mock.__aenter__))
```

Comment:
Maybe check `self.assertIsInstance(mock.__aexit__, AsyncMock)` too?

All these tests should also be applied to `AsyncMock`.